.github/workflows/deploy_to_environment.yml

```diff
@@ -71,26 +52,34 @@ jobs:
         if: contains(inputs.env, 'impl') || contains(inputs.env, 'prod')
         run: |
           ./scripts/rds-snapshot-app-db
-      - name: Run migrations
+      - name: Run database migrations
         env:
-          AWS_ACCOUNT_ID: ${{ secrets.AWS_ACCOUNT_ID }}
           ECR_REGISTRY: ${{ secrets.ECR_REGISTRY }}
           ECR_REPOSITORY: easi-db-migrate
-          LAMBDA_VERSION: ${{ inputs.lambda_version }}
-          ENV: ${{ inputs.env }}
+          NEW_IMAGE_TAG: ${{ env.GIT_HASH }}
+          TASK_FAMILY: "db-migrate-${{ inputs.env }}"
+          ECS_CLUSTER: "${{ inputs.env }}-easi-app"
+          SECURITY_GROUP_NAME: "ecs-db-migrate-${{ inputs.env }}"
```

Comment:
![image](https://github.com/CMSgov/easi-app/assets/15203744/21b107d5-4379-4d0f-a991-3f3f26db4456)

And this to 
```suggestion
          SECURITY_GROUP_NAME: "ecs-easi-app-db-migrate-${{ inputs.env }}"
```